.ci.yaml

```diff
@@ -58,11 +58,9 @@ platform_properties:
       caches: >-
         [
           {"name":"flutter_cocoapods","path":"cocoapods"},
-          {"name":"osx_sdk_13a233_13_15","path":"osx_sdk"},
+          {"name":"osx_sdk_13a233","path":"osx_sdk"},
```

Comment:
@godofredoc do you think this is the right thing to do here to avoid the caches from colliding?  Hopefully we don't have that problem once you are done the symlinking fixes.

Reply:
If I understand mac_toolchain correctly we don't need to differentiate between xcode and runtimes because they use different caches. In other words once the symlink changes land we will be able to keep a single cache per xcode version and single caches per runtime.